dw-snackbar.js

```diff
@@ -180,7 +180,7 @@ export class DwSnackbar extends LitElement {
             ${!toast.actionButton ? '' : this._getActionButtonTemplate(toast)}
 
             <!-- Dismiss button -->
-            ${toast.hideDismissBtn || toast.actionButton ? '' : html`
```

Comment:
I don't understand what's this change is?

Reply:
When actionButton is provided, close button was not shown. I think close button should be hidden based on only "hideDismissBtn" property not on "actionButton".

Reply:
That seems ok to me as per their spec. Do we need to show Action button & Close button both together?